docs/source/references/glossary.md

```diff
@@ -1,133 +1,84 @@
-# Glossary
+# Table of contents
+
+- [Glossary](#glossary)
+  - [Asset](#asset)
+  - [Byzantine fault-tolerance (BFT) {#bft}](#byzantine-fault-tolerance-bft-bft)
+  - [Leader](#leader)
+  - [World state view (WSV) {#wsv}](#world-state-view-wsv-wsv)
+  - [View change](#view-change)
+  - [Iroha Query](#iroha-query)
+  - [Iroha Special Instruction (ISI) {#isi}](#iroha-special-instruction-isi-isi)
+  - [Iroha Components](#iroha-components)
+  - [Iroha modules {#mod}](#iroha-modules-mod)
 
-//TODO: add links to docs.rs
+# Glossary
 
 Definitions of all Iroha-related entities can be found here.
 
-# Asset
+## Asset
 
 A representation of a valuable object on the blockchain.
 
 
-
-## Byzantine Fault-tolerance (BFT)
-
-The property of being able to properly function with a network
-containing a certain percentage of malicious actors. Iroha is capable
-of functioning with up to 33% malicious actors in its peer-to-peer
-network.
+## Byzantine fault-tolerance (BFT) {#bft}
```

Comment:
Is `{#xxx}` some kind of metadata? It will be displayed though

Reply:
This is how you normally define a custom heading for references. 

https://uhded.com/add-id-header-markdown

GitHub doesn't render it properly though,